platform-dependent-components/problem-solver/cxx/identifiers-module/agent/translate_main_system_idtfs_from_sc_to_file_agent.cpp

```diff
@@ -38,24 +30,27 @@ SC_AGENT_IMPLEMENTATION(TranslateMainSystemIdtfsFromScToFileAgent)
   while (iterator3PtrEdgeBelongsToNrelSystemIdtf->Next())
   {
     edgeBelongsToNrelSystemIdtf = iterator3PtrEdgeBelongsToNrelSystemIdtf->Get(2);
-    sourceOfEdgeBelongsToNrelSystemIdtf = m_memoryCtx.GetEdgeSource(edgeBelongsToNrelSystemIdtf);
+    sourceOfEdgeBelongsToNrelSystemIdtf = m_context.GetEdgeSource(edgeBelongsToNrelSystemIdtf);
     try
     {
-      systemIdentifier = GetSystemIdtfAndVerifyNode(m_memoryCtx, sourceOfEdgeBelongsToNrelSystemIdtf);
-      mainIdentifier = GetMainIdtfAndVerifyNode(m_memoryCtx, sourceOfEdgeBelongsToNrelSystemIdtf);
+      systemIdentifier = GetSystemIdtfAndVerifyNode(m_context, sourceOfEdgeBelongsToNrelSystemIdtf);
+      mainIdentifier = GetMainIdtfAndVerifyNode(m_context, sourceOfEdgeBelongsToNrelSystemIdtf);
       stringType = GetStrScType(sourceOfEdgeBelongsToNrelSystemIdtf);
 
       if (!systemIdentifier.empty() && !mainIdentifier.empty() && !stringType.empty())
       {
-        streamIdtfs << "{\"" << mainIdentifier << "\", "
-                    << "{\"" << systemIdentifier << "\", \"" << stringType << "\"}},\n";
+        streamIdtfs << R"({")" << mainIdentifier << R"(", )"
+                    << R"({")" << systemIdentifier << R"(", ")" << stringType << R"("}},\n)";
```

Comment:
everything is printed in one line with \n between entries, I guess previously they were separated with newline
![image](https://github.com/user-attachments/assets/d9b5ce10-a3ba-472e-a9a7-68c4befd9622)